### ISSUE TITLE
Remove findLast and findLastIndex from std/collections

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno_std#1525]
This feature is landing as a native JS feature in V8 9.7. https://v8.dev/blog/v8-release-97#findlast-and-findlastindex-array-methods

**Comment by LionC** [comment]
Let's do that once that is released!

**Comment by getspooky** [comment]
I agree with @LionC

**Comment by lucacasonato** [comment]
@LionC Will be in 1.16 on Tuesday.

**Comment by kt3k** [comment]
How about deprecating first before removing?

**Comment by getspooky** [comment]
The function is still available using native JS feature in `V8 9.7`, I suggest that we add alert message in the documentation for example:
```txt
`findLastIndex` is available as native function in javascript. learn more https://v8.dev/blog/v8-release-97#findlast-and-findlastindex-array-methods
```

**Comment by erfanium** [comment]
@lucacasonato i guess this issue should be closed
